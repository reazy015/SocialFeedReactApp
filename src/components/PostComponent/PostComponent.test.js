import React from 'react';
import MemoPost from './PostComponent';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import axios from 'axios';

jest.mock('axios');

describe('Post component', () => {
    it('renders correctly', async () => {
        const post = {
            id: 99,
            created_at: 'Fri Dec 29 19:15:04 +0000 2017',
            user: {
                name: 'John Doe'
            }
        }
        const component = shallow(<MemoPost post={post}/>);
        expect(toJson(component)).toMatchSnapshot();
    });
});

