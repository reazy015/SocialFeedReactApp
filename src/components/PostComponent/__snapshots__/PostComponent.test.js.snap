// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Post component renders correctly 1`] = `
<div
  className="feeds-item"
>
  <div
    className="feeds-item__date"
  >
    <strong>
      Posted at:
    </strong>
     
    <t
      ago={false}
      calendar={false}
      decimal={false}
      element={null}
      filter={[Function]}
      format="YYYY/MM/DD HH:mm"
      fromNow={false}
      interval={60000}
      local={false}
      onChange={[Function]}
      titleFormat=""
      toNow={false}
      unit={null}
      unix={false}
      utc={false}
      withTitle={false}
    >
      2017-12-29T19:15:04.000Z
    </t>
  </div>
  <div
    className="feeds-item__name"
  >
    <strong>
      By: 
    </strong>
    John Doe
  </div>
  <div
    className="feeds-item__text"
  />
</div>
`;
